web/content/release-3.1.adoc

```diff
@@ -24,5 +24,7 @@ Use the top menu bar for navigation in the following guides:
 * link:/{FOREMAN_VER}/Content_Management_Guide/index-katello.html[Content Management Guide]
 * link:/{FOREMAN_VER}/Configuring_Ansible/index-foreman-el.html[Configuring Foreman to use Ansible]
 * link:/{FOREMAN_VER}/Managing_Hosts/index-foreman-el.html[Managing Hosts Guide]
+* link:/{FOREMAN_VER}/doc-Managing_Configurations_Puppet/index-foreman-el.html[Configuring Hosts Using Puppet]
+* link:/{FOREMAN_VER}/doc-Managing_Configurations_Puppet/index-katello.html[Configuring Hosts Using Puppet]
```

Comment:
```suggestion
* link:/{FOREMAN_VER}/Managing_Configurations_Puppet/index-foreman-el.html[Configuring Hosts Using Puppet]
* link:/{FOREMAN_VER}/Managing_Configurations_Puppet/index-katello.html[Configuring Hosts Using Puppet]
```

Reply:
oh god I thought I deleted that doc- tag. 
Yeah, you're the Debian expert. I will add.